### PR TITLE
Fix: invalid URLs

### LINKED ISSUE #197: Task inspector generates invalid URLs for task definitions
It just sent me to `queue.http://localhost:3050/queue/v1/task/OSAG5pgDR9-ooA1NtWAdUQ`

### PR DESCRIPTION
**Pull request by helfi92**
Fixes #197.

### CODE REVIEW

File: .neutrinorc.js [hunk 1]
```diff
@@ -75,29 +75,29 @@ module.exports = {
         favicon: './logo.png',
       },
       devServer: {
-        port: process.env.PORT || 9000,
+        port: process.env.PORT,
         historyApiFallback: { disableDotRule: true },
         proxy: {
           '/login': {
-            target: process.env.TASKCLUSTER_ROOT_URL,
+            target: `http://localhost:3050`,
```
Comment (eliperelman) [inline .neutrinorc.js]: Use single quotes.

File: .neutrinorc.js [hunk 2]
```diff
@@ -75,29 +75,29 @@ module.exports = {
...
       },
       env: {
         APPLICATION_NAME: 'Application Name',
         LOGIN_STRATEGIES: '',
-        PORT: 9000,
-        TASKCLUSTER_ROOT_URL: 'http://localhost:3050',
-        GRAPHQL_SUBSCRIPTION_ENDPOINT: '',
-        GRAPHQL_ENDPOINT: '/graphql',
+        PORT: 5080,
+        TASKCLUSTER_ROOT_URL: `https://taskcluster.net`,
```
Comment (eliperelman) [inline .neutrinorc.js]: Use single quotes.

File: src/utils/validateActionsJson.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-let validateActionsJson = null;
+let actionsJsonSchemaResponse;
+let validateActionsJson;
 
 export default async () => {
   if (!validateActionsJson) {
     actionsJsonSchemaResponse = await fetch(
       schema(
-        `https://${process.env.TASKCLUSTER_ROOT_URL}`,
+        process.env.TASKCLUSTER_ROOT_URL,
```
Comment (eliperelman) [inline src/utils/validateActionsJson.js]: Any reason to not use `urls.schema` here instead of re-using the environment variable?

Comment (helfi92) [reply src/utils/validateActionsJson.js]: I should definitely use `urls.schema`. Thank you sir 💯

**Pull request merged**